java-symbolic-execution/src/test/files/se/Spring5Assert.java

```diff
@@ -0,0 +1,160 @@
+package symbolicexecution.behaviorcache;
+
+import java.util.Collection;
+import java.util.Map;
+import java.util.function.Supplier;
+import org.springframework.lang.Nullable;
+import org.springframework.util.Assert;
+
+class Spring5Assert {
+
+  Supplier<String> func = () -> "message";
+
+  void testNotNull(@Nullable Object o) {
+    Assert.notNull(o, () -> "message");
+    o.toString();
+  }
+
+  void testNotNullNc(@Nullable Object o) {
+    o.toString(); // Noncompliant
+  }
+
+  void testNotNull2(@Nullable Object o) {
+    Assert.notNull(o, func);
+    o.toString();
+  }
+
+  void testNotNull2Nc(@Nullable Object o) {
+    o.toString(); // Noncompliant
+  }
```

Comment:
Same code than line 18-20, so useless. It does not test anything different. A variant could be:
```java
void testNotNull2(@Nullable Object o) {
    Assert.notNull(o, func);
    if (o != null) { // Noncompliant
      // do something
    }
  }
```